perk-store-webapps/src/main/webapp/WEB-INF/jsp/perkstoreOrder.jsp

```diff
@@ -4,12 +4,20 @@
 <%@ page import="org.exoplatform.container.ExoContainerContext"%>
 <%@ page import="org.exoplatform.services.resources.ResourceBundleService"%>
 <%@ page import="org.exoplatform.perkstore.service.PerkStoreService"%>
+<%@ page import="org.exoplatform.portal.application.PortalRequestContext"%>
+<%@ page import="org.exoplatform.web.application.RequestContext"%>
 <%
   String title = "My orders";
   String titleOrders = "Orders";
   String titlePerkStore = "Perk Store";
   String textPerkStoreWidget = "Shop now !";
+  String perkstoreUrl = "" ;
+  String myOrdersUrlUrl = "";
+
   try {
+    PortalRequestContext requestContext = ((PortalRequestContext) RequestContext.getCurrentInstance());
+    perkstoreUrl = "/portal/" + requestContext.getPortalOwner() + "/perkstore";
```

Comment:
Could you replace it with `org.exoplatform.social.core.service.LinkProvider#getPortalName` to replace the string *portal* too